### PR TITLE
Adopt usage of lombok

### PR DESCRIPTION
**Pull request by martijn9612**

### CODE REVIEW

**Comment by TimvdLippe** [comment]
LGTM :+1:

File: src/main/java/ch/wisv/areafiftylan/web/controller/WebRestController.java [hunk 1]
```diff
@@ -21,13 +20,15 @@
     TournamentServiceImpl tournamentService;
     EventServiceImpl eventService;
     SponsorServiceImpl sponsorService;
+    CommitteeMemberServiceImpl committeeService;
```
Comment (skamoen) [inline src/main/java/ch/wisv/areafiftylan/web/controller/WebRestController.java]: This is not how you're supposed to use Services in the controller. Services have an interface, and an implementation and you declare the interface as a field in your controller. A service for each web entity might be a little overkill, zo maybe combine these into two or three services (Something like EventService for tournaments, events and other... events) and a Webservice for other static content such as sponsors, committees and such.

**Comment by skamoen** [comment]
The Lombok part looks good, but I would make a seperate PR for the web part.

**Comment by martijn9612** [comment]
I have made a mistake apparently, the first commit should not be in this pull request. My bad. I will remove the pull request and open a correct one with the issues fixed.

**Pull request closed without merging**